content/en/security/application_security/enabling/compatibility/serverless.md

```diff
@@ -0,0 +1,73 @@
+---
+title: Serverless Compatibility Requirements 
+kind: documentation
+code_lang: serverless
+type: multi-code-lang
+code_lang_weight: 90
+---
+
+<div class="alert alert-info">ASM support for AWS Lambda is in beta. Threat detection is done by using the Datadog's lambda extension.<br><br> Don’t see your desired technology listed here? Datadog is continually adding additional support. Fill out <a href="https://forms.gle/gHrxGQMEnAobukfn7">this short form to send us details</a>.</div>
+
+## Language and framework compatibility
+
+### Supported cloud environments
+
+- AWS Lambda (beta)
+
+### Version dependencies
+
+- Lambda Extension version: `39`
+- Serverless plugin version: `5.20.0`
+
+### Supported languages and their requirements
+
+Node
+: If you are bundling using webpack or esbuild, [mark the Datadog libraries as external][4].
+
+Python
+: 
+
+Java
+: To fully instrument your serverless application with distributed tracing, your Java Lambda functions must use the Java 8 Corretto (`java8.al2`) or Java 11 (`java11`) runtimes with at least 1024MB of memory.
+: If you use the Datadog Lambda layers `dd-trace-java:4` (or older) and `Datadog-Extension:24` (or older), follow the instructions in [Upgrade Instrumentation for Java Lambda Functions][3].
+
+Go
+: 
+
+.NET
+: 
+
+### ASM capabilities 
+The following ASM capabilities are not supported for Lambda functions:
+ - ASM Risk Management
+ - IP, user, and suspicious request blocking
+ - 1-Click enabling ASM
+
+## ASM capabilities support
+
+The following ASM capabilities are supported for Serverless, for the specified Datadog lambda extension version:
+
+| ASM capability                   | Minimum extension version |
+| -------------------------------- | ----------------------------|
+| Threat Detection <br/> --> Business logic API  | x.x <br/>x.x   |
```

Comment:
Threat Monitoring NOT SUPPORTED for Ruby, PHP. For everything else, supported

Version support:
- Lambda Extension version: 39
- Serverless plugin version: 5.20.0
- Business logic capabilities follow the language-specific versions in which the service is built

Reply:
```suggestion
| Threat Detection <br/> --> Business logic API  | Supported for Node.js, Java, Python, Go, and .NET with Lambda Extension version 39 and Serverless plugin version 5.20.0. <br/> --> Business logic capabilities follow the language-specific versions in which the service is built. |
```